src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageViewModel.cs

```diff
@@ -296,6 +296,38 @@ public bool HasLaunchOption
             }
         }
 
+        public bool NativeCodeDebugging
+        {
+            get
+            {
+                if (!IsProfileSelected)
+                {
+                    return false;
+                }
+
+                if (SelectedDebugProfile.OtherSettings.Keys.Contains("nativeDebugging"))
```

Comment:
ContainsKey would be better:
```suggestion
                if (SelectedDebugProfile.OtherSettings.ContainsKey("nativeDebugging"))
```